DXRModules/DeusEx/Classes/DXREnemyRespawn.uc

```diff
@@ -27,19 +27,22 @@ struct OriginalEnemy {
     var bool bReactFutz, bReactPresence, bReactLoudNoise, bReactAlarm, bReactShot, bReactCarcass, bReactDistress, bReactProjectiles;
     var int time_died;
 };
+
 var OriginalEnemy enemies[256];
 var int time;
 var config name dont_respawn[8];
+var #var(DeusExPrefix)Carcass carcs[256];
```

Comment:
maybe we should create a new separate DXRZombies.uc file? these classes that carry state and aren't transient can sometimes cause issues with backwards compatibility, so maybe it's best to keep them smaller and separate? I also don't see any shared logic so maybe no downside to a new file

Reply:
Do you mean that DXREnemyRespawn would just be for respawning, and DXRZomies would be for zombie reanimation?

Reply:
yea, not that this file is out of control, but there isn't really any reason they should be in the same file